nox/popen.py

```diff
@@ -21,16 +21,19 @@
 from typing import IO, Mapping, Sequence
 
 
-def shutdown_process(proc: subprocess.Popen) -> tuple[bytes, bytes]:
+def shutdown_process(
+    proc: subprocess.Popen,
+    interrupt_timeout: float,
+    terminate_timeout: float,
```

Comment:
Let's allow waiting without timeout as well.

```suggestion
    interrupt_timeout: float | None,
    terminate_timeout: float | None,
```